cpp/inc/bond/ext/grpc/unary_call.h

```diff
@@ -100,28 +101,35 @@ namespace detail {
         }
     };
 
-}
+} // namespace detail
 
 /// @brief The details of a single async, unary call.
 ///
 /// Call \p Finish or \p FinishWithError to send a response back to the
 /// client.
 ///
 /// If no explicit call to Finish/FinishWithError has been made before this
-/// unary_call instance is destroyed, a generic internal server error is
-/// sent.
+/// unary_call instance is destroyed or moved from, a generic internal
+/// server error is sent.
```

Comment:
Should we add a comment on the codegen on the server stub similar to this one where we define the expected behavior of the life of the unary_call that is received?

Reply:
If we start lifting documentation to the generated code, we're going to get a lot of duplication, so I'd rather not say anything, hopefully causing a search for (and a finding of) unary_call's documentation.